std_msgs/Header         header

# The name of the link for which we are computing IK
string link_name

# The pose of the       link
geometry_msgs/Pose      pose

---
duration                duration

# True if the ik was found
bool                    found_ik

# If false, something went wrong with the kinematics
bool                    found_group

# The returned metrics
sensor_msgs/JointState  joint_states
geometry_msgs/Pose      estimated_pose
geometry_msgs/Pose      estimation_error
float64                 manipulability
float64                 manipulability_index
float64                 max_payload
